client/src/components/Header/header.css

```
@@ -118,6 +118,16 @@ li.user-state-link > a:focus {
     left: 10%;
   }
 }
+/* hits */
+.ais-Highlight-highlighted {
```

Try something more specific to remove `!important`
```suggestion
.fcc_searchBar .ais-Highlight-highlighted {
```